buildlib/tools/builds.sh

```diff
@@ -0,0 +1,428 @@
+#!/bin/bash -eExl
+
+realdir=$(realpath $(dirname $0))
+source ${realdir}/common.sh
+source ${realdir}/../az-helpers.sh
+
+#
+# Build documentation
+#
+build_docs() {
+	doxy_ready=0
+	doxy_target_version="1.8.11"
+	doxy_version="$(doxygen --version)" || true
+
+	# Try load newer doxygen if native is older than 1.8.11
+	if ! (echo $doxy_target_version; echo $doxy_version) | sort -CV
+	then
+		if az_module_load tools/doxygen-1.8.11
+		then
+			doxy_ready=1
+		fi
+	else
+		doxy_ready=1
+	fi
+
+	if [ $doxy_ready -eq 1 ]
+	then
+		echo " ==== Build docs only ===="
+		../configure --prefix=$ucx_inst --with-docs-only
+		make_clean
+		$MAKE  docs
+		make_clean # FIXME distclean does not work with docs-only
+	fi
+}
+
+#
+# Building java docs
+#
+build_java_docs() {
+	echo " ==== Building java docs ===="
+	if az_module_load dev/jdk && az_module_load dev/mvn
+	then
+		../configure --prefix=$ucx_inst --with-java
+		$MAKE -C ../build-test/bindings/java/src/main/native docs
+		module unload dev/jdk
+		module unload dev/mvn
+	fi
+}
```

Comment:
should be part of jucx job
@petro-rudenko is it done already there?

Reply:
Yes, it's a part of AZP

Reply:
@avildema so let's remove it from here